src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core.Metric/MetricNames.cs

```diff
@@ -562,6 +563,7 @@ public static string GetSupportabilityAgentTimingMetric(string suffix)
         public const string SupportabilityEventHarvestTransactionEventHarvestLimit = SupportabilityEventHarvestPs + "AnalyticEventData" + SupportabilityEventHarvestHarvestLimit;
 
 
+        // TODO: stop generating new string
```

Comment:
This is a bit nitpicky, but I think we should remove the TODOs and just make a story for them if its needed.

Reply:
Ok, I'll pull them out. These comments were added since the concern was raised during our group code review about generating strings dynamically for metric names as an important thing that I needed to change in my own code.

Reply:
Fixed, it also conforms to the new naming spec, and follows existing patterns